### DISCUSSION TITLE
出现问题

### DISCUSSION THREAD

**Discussion by ytAlbert** [LFTEC/LearningFucker#29] [Q&A]
![image](https://github.com/LFTEC/LearningFucker/assets/43956694/61caed66-979d-48ca-8766-2b083f5cc371)

**Comment by LFTEC** [comment]
你是不是用的还是旧版本？
更新一下2.01版本


lsha ***@***.***> 于 2023年8月19日周六 20:37写道：

**Reply by ytAlbert** [reply]
已成功解决，感谢。

**Comment by LFTEC** [comment]
我已经修复该问题，现在好了，但是还是尽快升级新版本，server.jcdev.cc服务很快会停用

puff jc ***@***.***> 于 2023年8月20日周日 14:37写道：
